stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -492,6 +492,29 @@ public struct Unsafe${Mutable}RawBufferPointer
 %  end # mutable
   }
 
+%if Mutable:
+  /// Exchanges the byte values at the specified offsets
+  /// in the memory region.
```

Comment:
Question: What is the difference in meaning between this “in the memory region” and the documentation for the other method which says, “of the buffer”?

Reply:
"memory region" is used regularly in the `UnsafeRawPointer` documentation, which I interpret as emphasis on the untyped nature of raw pointers; I didn't use it well here.